compiler/passes/checkParsed.cpp

```diff
@@ -450,8 +450,8 @@ checkFunction(FnSymbol* fn) {
 static void checkOperator(FnSymbol* fn) {
   if (!fn->hasFlag(FLAG_OPERATOR) && !fn->hasFlag(FLAG_METHOD)) {
```

Comment:
I'm assuming that something like `proc R.==(a, b)` didn't do anything before (meaning - it wasn't actually called by `==`). But was it definable? Should it still produce the deprecation ?

Reply:
`proc R.==(a, b)` was an error prior to adding support for operator methods and operator methods are only defined with the operator keyword, so no one should have been able to write it

Reply:
Looks like in deleting duplicates it deleted the whole comment.  Anyways, `proc R.==(a, b)` was not possible before and in fact generated an error message, so I don't think it needs to be deprecated